src/fontMetricsData.js

```diff
@@ -1,3 +1,4 @@
+// @flow
 module.exports = {
```

Comment:
I recommend giving it an explicit tuple `[number, number, number, number]` type instead of an array type `number[]`. This allowed me to catch a (mostly harmless) error in `fontMetrics.js`. 
E.g. via a local type:
```
type FontMetrics = {[string]: [number, number, number, number]};

// character code => [depth, height, italic, skew]
const fontMetricsData: {[string]: FontMetrics} = {
...
};

export default fontMetricsData;
```